static/js/message_fetch.js

```diff
@@ -17,7 +17,9 @@ var consts = {
 function process_result(data, opts) {
     var messages = data.messages;
 
-    $('#connection-error').removeClass("show");
+    if (!($('#connection-error').hasClass('get-events-error'))) {
+        $('#connection-error'). removeClass("show");
```

Comment:
Why isn't this `ui_report.hide_error(...)`?

As a sidenote, I'm surprised our whitespace linter didn't catch the weird spacing here; want to look for whatever the eslint rule is for that?